source/agora/node/Ledger.d

```diff
@@ -236,6 +236,8 @@ public class Ledger
         // read back and cache the last block
         if (!this.storage.readLastBlock(this.last_block))
             assert(0);
+
```

Comment:
The commit message is unclear. "call the in"?

Maybe just write: "Clean expired enrollments after the block was finalized"

Reply:
The `git commit -m` command removes the text between two grave accents(`) in my commit message. I will change it. Thanks.